backend/danswer/danswerbot/slack/utils.py

```diff
@@ -41,7 +47,41 @@
 logger = setup_logger()
 
 
-DANSWER_BOT_APP_ID: str | None = None
+_DANSWER_BOT_APP_ID: str | None = None
```

Comment:
Think we should be avoiding usage of globals here as this isn't going to generalize or be reusable easily.